src/query/expression/src/kernels/sort.rs

```diff
@@ -41,9 +41,11 @@ use crate::Scalar;
 
 pub type AbortChecker = Arc<dyn CheckAbort + Send + Sync>;
 
+pub struct AbortError;
```

Comment:
So, do we need to create a dummy empty struct to trace the stack?


Can we just use '()' ?

Reply:
> So, do we need to create a dummy empty struct to trace the stack?

It's recommended to do so.

> Can we just use '()' ?

yes, addressed.